js/app.js

```diff
@@ -21,18 +21,15 @@ $(document).on('ready', function() {
     //console.log(playersArr);
   }
 
-  //reset keypress counter for each player (set marker back to start position)
+  $('#reset').on('click', function(){
+    resetPlayers();
+  });
 
-
-  //generates random Avater, player image
-  function randAvatar(input){
-    //can hard code in playerbuild for loop
-    //api to search spotify album covers for avater
-  }
-
-  //generates random token
-  function randToken(){
-    //can hard code in player for loop
+  // reset all players' pressNums to 0
```

Comment:
The resetPlayers function below does all the work, but you can tell on line 31 it relies on each player having a pressNum instance variable.